backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlDatabase.java

```diff
@@ -616,6 +616,24 @@ public DSLContext getJooq() {
     return jooq;
   }
 
+  void getJooqAsAdmin(JooqTransaction transaction) {
+    if (inTx()) {
+      String user = connectionProvider.getActiveUser();
+      try {
+        connectionProvider.setActiveUser(ADMIN_USER);
+        transaction.run(getJooq());
+      } finally {
+        connectionProvider.setActiveUser(user);
+      }
+    } else {
+      final Settings settings = new Settings().withQueryTimeout(TEN_SECONDS);
```

Comment:
can be moved up to class level ( or use existing) , no need to recreate object on each method innovation

Reply:
true

Reply:
done

Reply:
done